demo/api/src/footer/generated/footer.resolver.ts

```diff
@@ -5,13 +5,13 @@ import { InjectRepository } from "@mikro-orm/nestjs";
 import { EntityManager, EntityRepository } from "@mikro-orm/postgresql";
 import { Args, Mutation, Query, Resolver } from "@nestjs/graphql";
 
+import { FooterScope } from "../dto/footer-scope";
 import { Footer } from "../entities/footer.entity";
-import { FooterContentScope } from "../entities/footer-content-scope.entity";
 import { FooterInput } from "./dto/footer.input";
 import { FootersService } from "./footers.service";
 
 @Resolver(() => Footer)
-@RequiredPermission(["pageTree"])
+@RequiredPermission(["footers"])
```

Comment:
This should still be pageTree

Reply:
This is an autogenerated file, need to find where this gets generated from.

Reply:
This is also footers in Comet Starter... Would this maybe get generated from the Service Name?

For example news also has "news" as requiredPermission and products has "products"

Reply:
You can change the permission in the API Generator: https://github.com/vivid-planet/comet/blob/main/demo/api/src/footer/entities/footer.entity.ts#L13

Reply:
Nice, didn't know ;) Updated!